### PR TITLE
mom integration test

### PR DESCRIPTION
**Pull request by kensipe**
## High Level Description

What features does this change enable? What bugs does this change fix? High-Level description of how things changed.

## Related Issues

  - [DCOS-18957](https://jira.mesosphere.com/browse/DCOS-18957) Adds a Marathon on Marathon (MoM) integration test.

## Checklist for all PR's

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:   It is a test :)
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

### CODE REVIEW

**Comment by kensipe** [comment]
rebased

File: packages/dcos-integration-test/extra/test_packaging.py [hunk 1]
```diff
@@ -149,3 +148,23 @@ def test_packaging_api(dcos_api_session):
...
+
+
+def test_mom_installation(dcos_api_session):
+    """Test the Cosmos installation of marathon on marathon (MoM)
+    """
+
+    install_response = dcos_api_session.cosmos.install_package('marathon')
+    data = install_response.json()
+
+    dcos_api_session.marathon.wait_for_deployments_complete()
```
Comment (timcharper) [inline packages/dcos-integration-test/extra/test_packaging.py]: Would be nice to confirm that the deployment won't complete until health checks pass. I'm pretty sure that is the case :)

File: packages/metronome/buildinfo.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "requires": ["java", "exhibitor"],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://downloads.mesosphere.com/maven/dcos/metronome/0.2.3/metronome-0.2.3.tgz",
-    "sha1": "ef42cb936a59360ecd0dda3a434f75e1007f3527"
+    "url": "http://downloads.mesosphere.io/metronome/snapshots/metronome-0.2.4.tgz",
```
Comment (mellenburg) [inline packages/metronome/buildinfo.json]: Whats wrong with https?

**Comment by mellenburg** [comment]
@mesosphere-mergebot bump-ee

**Comment by kensipe** [comment]
great!  thx!!

File: packages/metronome/buildinfo.json (hunk 2 above)
Comment (kensipe) [reply packages/metronome/buildinfo.json]: I wasn't aware we have a preference... I pasted what I was working with locally.   I can default to https.

Comment (orsenthil) [reply packages/metronome/buildinfo.json]: defaulting to `https` better.

**Comment by orsenthil** [comment]
@kensipe , @timcharper , @ichernetsky  and @mellenburg  - please address this comment too: https://github.com/mesosphere/dcos-enterprise/pull/1563#issuecomment-336556025

When reviewing, please look at the EE PR and review/approve that one at the same time.

**Comment by mellenburg** [comment]
New test looks solid :)
https://teamcity.mesosphere.io/project.html?projectId=DcOs_Open_Test_IntegrationTest&testNameId=6971838291720178997&tab=testDetails

**Comment by kensipe** [comment]
@orsenthil thank you!   I have addressed the dcos-enterprise comment!

File: packages/metronome/buildinfo.json (hunk 2 above)
Comment (kensipe) [reply packages/metronome/buildinfo.json]: yeah.. this was a mistake... good catch @orsenthil the metronome bump was suppose to be on the other PR.   Let me update this again.  This should be just the new mom test.

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
I'd like to remove the ship-it label because the Pr is not ready yet.

We need to address these status checks.

img

**Comment by kensipe** [comment]
looking into it

**Comment by kensipe** [comment]
@lingmann @mellenburg can we fix this:  ` Failed to locate jps tool. No thread dump information is available`

**Comment by kensipe** [comment]
The tests are failing on a strict cluster...  I don't believe we support mom on strict.   We expect MoM-EE on strict.

**Comment by kensipe** [comment]
@mesosphere-mergebot bump-ee

**Comment by jgehrcke** [comment]
Ship-it was applied prematurely. See DCOS-19252.

**Comment by jgehrcke** [comment]
@kensipe can you revive this? I know all of this is a lot of pain, but the PR owner needs to be the driving force.

**Comment by kensipe** [comment]
@jgehrcke on it!  thanks!
I rebased to the latest on master.

**Comment by kensipe** [comment]
@mesosphere-mergebot bump-ee

**Comment by kensipe** [comment]
rebased again...

**Comment by kensipe** [comment]
@mesosphere-mergebot bump-ee

**Comment by kensipe** [comment]
rebased to master

**Comment by kensipe** [comment]
@mesosphere-mergebot bump-ee

**Comment by kensipe** [comment]
reviving this PR... will rebase and try again

**Comment by kensipe** [comment]
@mesosphere-mergebot bump-ee

**Comment by kensipe** [comment]
@mesosphere-mergebot bump-ee

**Comment by meichstedt** [comment]
@kensipe I think the milestone should be changed to 1.12 ;)

**Comment by orsenthil** [comment]
Applying the ship-it label manually on this revived PR.  Changes were requested by @lingmann when he was the administrator, now since he is not, I cannot seem to dismiss his changes requested state.

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/arm  https://jira.mesosphere.com/browse/DCOS-39852

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/arm https://jira.mesosphere.com/browse/DCOS-40611

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status mergebot/review/approved/min_2 https://jira.mesosphere.com/browse/DCOS-40612

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ship-it

**Pull request merged**